setup.py

```diff
@@ -83,7 +83,7 @@ def check_include(library_name, header):
     platforms=['any'],
     packages=packages,
     ext_modules=ext_modules,
-    install_requires=['aiodns>=1.0', 'pyasn1', 'pyasn1_modules', 'aiohttp'],
+    install_requires=['aiodns>=1.0', 'pyasn1', 'pyasn1_modules', 'aiohttp', 'lxml'],
```

Comment:
I'll need to test a bit, due to the differences between ElementTre and lxml's etree (https://github.com/lxml/lxml/blob/master/doc/compatibility.txt although there are mostly added features)